addons/account/models/partner.py

```diff
@@ -499,7 +499,7 @@ def create(self, vals_list):
     def _increase_rank(self, field):
         if self.ids and field in ['customer_rank', 'supplier_rank']:
             try:
-                with self.env.cr.savepoint():
+                with self.env.cr.savepoint(flush=False):
```

Comment:
@nim-odoo this is fixing the issue but... I don't understand exactly what is happening...
Creating a new savepoint will call `flush` that is triggering some methods in `mail.thread` and then... :boom: